nemo/collections/tts/models/fastpitch.py

```diff
@@ -457,8 +509,11 @@ def validation_epoch_end(self, outputs):
         self.log("val_mel_loss", mel_loss)
         self.log("val_dur_loss", dur_loss)
         self.log("val_pitch_loss", pitch_loss)
+        if outputs[0]["energy_loss"] is not None:
+            energy_loss = collect("energy_loss")
+            self.log("val_energy_loss", energy_loss)
 
-        _, _, _, _, spec_target, spec_predict = outputs[0].values()
+        _, _, _, _, _, spec_target, spec_predict = outputs[0].values()
```

Comment:
we need to be careful about the returned tuple because the order of items matters. We had bugs before and training process was blocked.

Reply:
Verified.